doc/rst/technotes/globalvars.rst

```diff
@@ -0,0 +1,47 @@
+.. _readme-globalvars:
+
+==================================
+Global Variables
```

Comment:
I think Brad objects to the phrase "global variables", but I don't know what else to call them.  (Or if these are actually outside the scope of the outermost module.)  And to nitpick, I think they're all params, not variables.  :)

Reply:
Should it be Global Params then?

Reply:
I think Brad mostly objects to calling symbols defined at module scope "global".  I think these could reasonably be called "global", though.

Reply:
Maam is the name 'Global Variables` fine then?

Reply:
I think it's fine.  Variables can be specifically those that are declared `var`, or they can also be the general category of symbols which includes `var`, `param`, etc.  At least, that's the impression I'm getting from the spec's [section on the subject](https://chapel-lang.org/docs/latest/language/spec/variables.html)